src/main/java/com/softserve/kickscootertrip/repository/TripRepository.java

```diff
@@ -0,0 +1,27 @@
+package com.softserve.kickscootertrip.repository;
+
+import com.softserve.kickscootertrip.model.entity.TripEntity;
+import org.springframework.data.geo.Point;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+
+import java.time.LocalDateTime;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+public interface TripRepository extends JpaRepository<TripEntity, Long> {
+List<TripEntity> findByUserId(UUID id);
+    public TripEntity findByUserIdAndStatus(UUID userId, String status);
+    public TripEntity findByTripId(UUID tripId);
+
+
+    @Modifying
+    @Query(value = "update trip t set t.point=?1, t.distance = ?2 where t.trip_id = ?3", nativeQuery = true)
```

Comment:
Why do you need this query to be native?

Reply:
There is an error "Caused by: java.lang.IllegalArgumentException: org.hibernate.hql.internal.ast.QuerySyntaxException: trip is not mapped [update trip t set t.point=?1, t.distance = ?2 where t.trip_id = ?3]"
Ok, I will find the way to fix it in other way :)